project/src/Controller/DomainNameController.php

```diff
@@ -111,4 +111,26 @@ public function delete(Request $request, DomainName $domainName, EntityManagerIn
 
         return $this->redirectToRoute('domain_name_index', [], Response::HTTP_SEE_OTHER);
     }
+
+    /**
+     * @param DomainName $domainName
+     * @param EntityManagerInterface $entityManager
+     * @return void
+     */
+    private function setBinder(DomainName $domainName, EntityManagerInterface $entityManager)
+    {
+        $siteClientToServicesBinder = new SiteClientToServicesBinder();
+        $siteClientToServicesBinder
+            ->setClient($domainName->getClient())
+            ->setSite($domainName->getSite() ?: null)
```

Comment:
Not necessary to use a ternary here ?
If getSite is null, the getter will return null no ?